pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: layout.pug
+navigationTitle: Explore New Cluster
+title: Explore New Cluster
```

Comment:
These don't look right to me...   You will also need to check the menuWeight when adding a new page.

Probably something like?

```suggestion
navigationTitle: EKS Cluster IAM Policies and Roles
title: EKS Cluster IAM Policies and Roles
```